frontend/src/utils/backgroundFilter/applyBackgroundFilter/applyBackgroundFilter.ts

```diff
@@ -0,0 +1,64 @@
+import { hasMediaProcessorSupport, Publisher, VideoFilter } from '@vonage/client-sdk-video';
+import { UserType } from '../../../Context/user';
+import { BACKGROUNDS_PATH } from '../../constants';
+import { setStorageItem, STORAGE_KEYS } from '../../storage';
+
+/**
+ * Applies a background filter to the publisher.
+ * @param {Publisher} publisher - The Vonage Publisher instance.
+ * @param {string} backgroundSelected - The selected background option.
+ * @param {Function} [setUser] - Optional function to update user state.
+ * @param {Function} [setBackgroundFilter] - Optional function to set background filter state.
+ * @param {boolean} [storeItem] - Optional flag to determine if the filter should be stored.
```

Comment:
we don't typically put in the parameters in square braces. can you please update it? thanks 🙏